### ISSUE TITLE
Twitter Block: Tweetstorm unroll button missing in IE11

### ISSUE THREAD

**Issue by aaronrobertshaw** [Automattic/jetpack#16371]
#### Steps to reproduce the issue

1. Edit a post using IE11 (tried on Windows 10)
2. Add a twitter block
3. Enter a URL for a tweet e.g. https://twitter.com/jensenharris/status/993695458517110784
4. Notice there is no "unroll" button present

#### What I expected
Expected the "unroll" button for importing a tweetstorm to be visible.

#### What happened instead
The "unroll" button is not displayed. It doesn't appear within the page markup either.

**Screenshots**
Actual:
<img width="476" alt="MissingUnrollButton" src="https://user-images.githubusercontent.com/60436221/86332881-760a8400-bc8e-11ea-9f56-194a6534e6c6.PNG">

Expected:
<img width="614" alt="Screen Shot 2020-07-02 at 6 04 52 pm" src="https://user-images.githubusercontent.com/60436221/86332984-9d615100-bc8e-11ea-9193-f6e409bb55f4.png">

**Comment by pento** [comment]
It looks like there's a dependency which is being bundled that isn't IE11 compatible.

I'm not super familiar with Jetpack's build system, but I appeared to be making a bit of progress by adding extra entries to the transpile list in `webpack.config.extensions.js`. I got as far as setting it to `exclude: /node_modules\/(?!punycode|query-string|split-on-first|strict-uri-encode)/` before I was unable to determine the next culprit.
